retrace-server.spec

```diff
@@ -20,6 +20,10 @@ BuildRequires: meson
 BuildRequires: python3-devel
 BuildRequires: texinfo
 BuildRequires: xmlto
+BuildRequires: gzip
+BuildRequires: lsof
+BuildRequires: tar
+BuildRequires: xz
```

Comment:
Just my OCD: Can you sort all the `BuildRequires` alphabetically?

Reply:
Sure, no problem.